internal/cli/config/config.go

```diff
@@ -51,11 +62,7 @@ func (opts *configOpts) IsOpsManager() bool {
 	return opts.Service == config.OpsManagerService
 }
 
-func (opts *configOpts) IsCloudManager() bool {
-	return opts.Service == config.CloudManagerService
-}
```

Comment:
[q] Was this just not being used or now exists somewhere else?

Reply:
not being used